app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,43 @@
+class CommentsController < ApplicationController
+  def create
+    @talk = Talk.find(params[:talk_id])
+    @comment = Comment.new(talk_id: @talk.id, content: params[:comment][:content], user_id: 1)
+    if @comment.save
+      flash[:notice] = 'Thanks for commenting'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.